### PR TITLE
*: Use the new way to handle the DDL operations

### PR DESCRIPTION
**Pull request by zimulala**
Remove the old code.

### CODE REVIEW

**Comment by hanfei1991** [comment]
LGTM

File: store/tikv/kv.go [hunk 1]
```diff
@@ -151,7 +151,7 @@ func NewMockTikvStoreWithCluster(cluster *mocktikv.Cluster) (kv.Storage, error)
 	mocktikv.BootstrapWithSingleStore(cluster)
 	mvccStore := mocktikv.NewMvccStore()
 	client := mocktikv.NewRPCClient(cluster, mvccStore)
-	uuid := fmt.Sprintf("mock-tikv-store-:%v", time.Now().Unix())
+	uuid := fmt.Sprintf("mock-tikv-store-%v-%v", time.Now().Unix(), rand.Intn(10))
```
Comment (hanfei1991) [inline store/tikv/kv.go]: increase it

**Comment by zimulala** [comment]
PTAL @tiancaiamao @coocood @shenli

**Comment by coocood** [comment]
LGTM

**Pull request merged**